crates/wasmtime/Cargo.toml

```diff
@@ -417,3 +420,5 @@ component-model-async-bytes = [
 # Enables support for guest debugging.
 debug = ['runtime']
 
+# Enables support for defining compile-time builtins.
+compile-time-builtins = ['dep:wasm-compose', 'dep:tempfile']
```

Comment:
Would it make sense to plumb this feature in more locations? For example the preexisting `expose_unsafe_intrinsics` isn't gated by this feature but arguably may want to be? I'm not sure how beneficial that would be though.